### ISSUE TITLE
How to generate a real landslide mode 

### ISSUE THREAD

**Issue by xuhenry** [TITAN2D/titan2d#4]
Dear All, 
   Titan2D is a powerful code for the simulation of the landslide. But how to generate a real landslide model with complex boundary. For I only see some examples with the simple shape, such as  cylinder and so on.
Thanks a lot!
Best regards,
Henry

**Comment by nsimakov** [comment]
Dear Henry,

Yes, Titan2D can only handle two simple forms for now. It shouldn't be very difficult to implement and we have discussed it earlier, but it is not clear when we will have time to add that.

Thanks,
Nikolay

**Comment by xuhenry** [comment]
Dear Nikolay,
    Many thanks for your reply. Furthermore, will you give me some suggestion about how to perform  a landslide with complex boundary?
   May I import the Piles information from a file?
Thanks a lot!
Best regards,
Henry

**Comment by nsimakov** [comment]
Yes, you can try to use a set of piles to approximate more complex geometry. 

The input file is a python script so you can add it within your input script. Basically you need to add piles multiple times with modified arguments for addPile method:

```python
piles_xy = [
    [644956.0, 2157970.0],
    [644936.0, 2157970.0]
]

for xy in piles_xy:
    sim.addPile(
        pile_type='Paraboloid',
        height=30.0,
        center=xy,
        radii=[55.0, 55.0],
        orientation=0.0,
        Vmagnitude=0.0,
        Vdirection=0.0,
)
```

The problem is that I don't think it will handle overlapping piles correctly, particularly the initial material volume. Current stable release has a bug in calculation of cylindrical piles volume by factor of two results are not identical but qualitatively somewhat similar (parabolic pile are ok and it is already fixed in master).

Regards,
Nikolay

**Comment by xuhenry** [comment]
Thanks a lot.
Yes, there may be overlapping of piles with cylindrical or parabolic piles. If the rectangle pile can be used, it will be ok for a landslide with complex shape  which can be discrete a series of rectangle piles.
Thanks  a lot!
Best regards,
Henry
